package.json

```diff
@@ -21,11 +21,12 @@
   "devDependencies": {
     "babel-core": "^6.3.15",
     "babel-loader": "^6.2.0",
+    "babel-plugin-transform-object-assign": "^6.3.13",
```

Comment:
I think that ideally, you'd fix version all of these instead of using ^. Just removing it should be enough.